interpreter/exec/v128.ml

```diff
@@ -2,4 +2,24 @@ include Simd.Make
   (struct
     include Bytes
     let bytewidth = 16
+
+    (* Constructs v128 of a certain shape from a list of strings. Does not do any validation,
```

Comment:
The other of_string functions do validation, so it would be more consistent to do that here as well (other callers might screw it up as well). The parser can simply catch the exception and generate an error.

Reply:
Makes sense, thanks for pointing this out. Moved the validation into v128.to_strings.
The error reporting for this is bad, defaults to the whole line, where it should pinpoint the constant that is out of bounds, and I can fix it in a subsequent patch.